src/main/java/io/stargate/sgv2/jsonapi/service/processor/CommandProcessor.java

```diff
@@ -63,8 +62,6 @@ public <T extends Command> Uni<CommandResult> processCommand(
               return Uni.createFrom().item(operation);
             })
 
-        // run on worker thread pool since operation hsa blocking code for vectorize
```

Comment:
what are the implications of removing this? what threads would this run on instead?

Reply:
The current vectorize (embedding service api) is implemented as blocking call. This is need for it. When we enable vectorize we need to change it as a reactive code.